main.py

```diff
@@ -19,7 +19,6 @@
 from cogs.help import HelpCommandCog
 from cogs.gov import GovCommandsCog
 
-
```

Comment:
See: https://github.com/BloomGameStudio/BloomDiscordBot/pull/107

Reply:
> See: #107

merged in